docs/whatsnew/1.0.0/1_0_0.rst

```diff
@@ -0,0 +1,70 @@
+
+The documentation was completely reworked.
+We separated :doc:`the tutorial section</tutorial/index>`, which teaches you how to use Pylake from :doc:`the theory section</theory/index>`, which teaches you the theory underlying the implemented methods.
+In addition, both sections were extended considerably.
+
+Every tutorial now downloads its own `example data <https://doi.org/10.5281/zenodo.7729525>`_ automatically when you run it.
+This allows you to immediately try out the various functionalities Pylake has to offer.
+The new :doc:`theory section</theory/index>` provides examples and references to literature for further reading.
+
+Creating kymographs from camera recordings
+------------------------------------------
+
+.. image:: kymo_to_stack.png
+
+A new feature introduced in `1.0.0` is the ability to convert camera recordings to kymographs.
+After defining a tether, you can call :meth:`~lumicks.pylake.ImageStack.to_kymo` which rotates the camera images such that the tether is aligned with the horizontal axis and sums a region around the tether.
+To read more about this feature, please refer to the :ref:`tutorial<kymo_from_image>`.
+
+Bias correction
+---------------
+
+When tracking kymographs, the default refinement algorithm used to achieve sub-pixel accuracy is based on centroid refinement.
+Centroid refinement can suffer from bias (tending towards the pixel center) when there is considerable background in the image.
+
+.. figure:: bias_correction.png
+
+    Bias due to high background.
+    On the x-axis is the true position of a fluorescent spot, while on the y-axis, we see its inferred location.
+    From left to right: regular centroid refinement, bias-corrected refinement and Gaussian refinement.
+    The step-like progression in the centroid case is due to the bias induced by the background intensity.
+    Note how the bias corrected variant does not tend to the pixel centers.
+
+Pylake `1.0.0` removes this bias using the method presented in :cite:`berglund2008fast`.
+For best results, we still recommend using :ref:`gaussian refinement<kymotracker_gaussian_refinement>` after tracking.
+
+Scale
+-----
+
+Camera images now show the image in microns rather than pixels.
+
+.. figure:: calibrated.png
+
+Sometimes, it can be nice to have a scale bar.
+Pylake `1.0.0` adds a plot widget that you can add to just about any plot.
+Simply add `scale_bar=lk.ScaleBar()` to the `.plot()` or `.export_video()` command and it will show a scale bar in the plot.
+It works in both movie export as well as regular plotting.
+See :class:`~lumicks.pylake.ScaleBar` for more information.
+
+.. figure:: scalebar.gif
+
+    Movie exported from Pylake with `scan.export_video("rgb", "scan_stack.gif", scale_bar=lk.ScaleBar()))`.
+
+Other changes
+-------------
+
+In addition to these functionalities, we implemented various other bug-fixes and improvements.
+For a full list of all the changes, please refer to the full :doc:`changelog</changelog>`.
```

Comment:
I wonder if we should note that there were extensive breaking changes? And/or perhaps explaining what `v1` actually means for people that aren't familiar with semver? Basically just mentioning that, from now, breaking changes lead to an increment in the major version number

Reply:
Yeah, we probably should. How about:
```
In addition to these functionalities, we implemented various other bug-fixes and improvements.
Since this is a major version release, it also includes some breaking changes.
Note that we adhere to `semantic versioning <https://semver.org/>`, meaning that we increment the major version number to indicate that there are breaking changes.
For a full list of all the changes, please refer to the full :doc:`changelog</changelog>`.
```